src/layers/models/layersManager.ts

```diff
@@ -130,25 +146,33 @@ export class LayersManager {
           throw new ConflictError(message);
```

Comment:
Don't we need to add here "handleSpanOnError"

Reply:
this code is inside a try-catch block, so i added `handleSpanOnError` inside the catch scope.